sessionctx/variable/tidb_vars.go

```diff
@@ -1186,7 +1186,7 @@ const (
 	DefTiDBTTLScanWorkerCount                              = 4
 	DefTiDBTTLDeleteWorkerCount                            = 4
 	DefaultExchangeCompressionMode                         = kv.ExchangeCompressionModeUnspecified
-	DefTiDBEnableResourceControl                           = false
```

Comment:
Now, We have a feature tag to make the default-disable feature run in all the tests.

Reply:
Here is the feature tag for distributed DDL.

https://github.com/pingcap/tidb/tree/master/sessionctx/variable/featuretag/distributereorg